shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -526,15 +612,13 @@ public boolean onTouchEvent(MotionEvent event) {
         final int kPointerDataFlagBatched = 1;
 
         int pointerCount = event.getPointerCount();
-
-        ByteBuffer packet = ByteBuffer.allocateDirect(pointerCount * kPointerDataFieldCount * kPointerBytesPerField);
-        packet.order(ByteOrder.LITTLE_ENDIAN);
-
         int maskedAction = event.getActionMasked();
         int pointerChange = getPointerChangeForAction(event.getActionMasked());
+        List<PointerAction> requiredActions = new ArrayList<PointerAction>();
```

Comment:
nit: I'd rename this. It's unclear what they are *required* for. Maybe just `actions`, or `actionsForFramework`.